Hazelcast.Net/Hazelcast.Core/IPNCounter.cs

```diff
@@ -0,0 +1,151 @@
+﻿/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Hazelcast.Client.Spi;
+
+namespace Hazelcast.Core
+{
+    /// <summary>
+    /// PN (Positive-Negative) CRDT counter.
+    /// </summary>
+    /// <remarks>
+    /// The counter supports adding and subtracting values as well as retrieving the current counter value.
+    /// Each replica of this counter can perform operations locally without coordination with the other replicas, 
+    /// thus increasing availability.
+    /// The counter guarantees that whenever two nodes have received the same set of updates, 
+    /// possibly in a different order, their state is identical, and any conflicting updates are merged automatically.
+    /// If no new updates are made to the shared state, all nodes that can communicate will eventually have the same data.
+    ///
+    /// The updates to this counter are applied locally when invoked on a CRDT replica. 
+    /// A replica can be any hazelcast instance which is not a client or a lite member. 
+    /// The number of replicas in the cluster is determined by the PNCounterConfig configuration value.
+    ///
+    /// When invoking updates from non-replica instance, the invocation is remote.
+    /// This may lead to indeterminate state - the update may be applied but the response has not been received.
+    /// In this case, the caller will be notified with a <exception cref="TargetDisconnectedException"/> when invoking from a client.
```

Comment:
Is this a proper usage of `<exception>` tag. I think we should use `<see>` tag instead.

Reply:
<exceptions> tag is just dedicated element for exceptions. <see> is more generic form.
I can change it to <see> if it's more consistent with rest of the documentation. 
In *.xml comment file we distribute with client lib it indicates exception that can be thrown

Reply:
Please let me know if it's better to use `<see`> so I will change it in such case...

Reply:
Fixed